junitparser/cli.py

```diff
@@ -0,0 +1,50 @@
+from argparse import ArgumentParser
+
+from . import JUnitXml, version
+
+
+def merge(paths, output):
+    """Merge xml report.
+    """
+    result = sum((JUnitXml.fromfile(path) for path in paths), JUnitXml())
```

Comment:
used `sum` to remove the AssignThenModify anti-pattern by using proper comprehensions 
was:
```py
    result = JUnitXml()
    for path in args.paths:
        result += JUnitXml.fromfile(path)
```